utils/scripts/install-centos8.sh

```diff
@@ -67,5 +68,20 @@ dnf install \
     python3-yaml \
     sg3_utils \
     valgrind-devel \
+    which \
     yasm
+
+echo "is fedora java? $fedora_java"
+
+if [ "$fedora_java" = "yes" ]; then
```

Comment:
Per above, use natural shell booleans:

```suggestion
if "$java"; then
```